python/dask_cudf/dask_cudf/tests/test_groupby.py

```diff
@@ -679,7 +695,7 @@ def test_groupby_agg_redirect(aggregations):
     ],
 )
 def test_is_supported(arg, supported):
-    assert _aggs_supported(arg, SUPPORTED_AGGS) is supported
+    assert _aggs_supported(arg, AGGS) is supported
```

Comment:
Nitpicky, but we might want to keep this as `SUPPORTED_AGGS` to make sure we don't eventually mess something up with support for new aggregations down the line:

```suggestion
    assert _aggs_supported(arg, SUPPORTED_AGGS) is supported
```